pkg/graph/schema.resolvers.go

```diff
@@ -1086,8 +1086,8 @@ func (r *mutationResolver) CreateTRBAdminNoteConsultSession(ctx context.Context,
 	return resolvers.CreateTRBAdminNoteConsultSession(ctx, r.store, input)
 }
 
-// CreateTRBAdminNoteAdviceLetter is the resolver for the createTRBAdminNoteAdviceLetter field.
-func (r *mutationResolver) CreateTRBAdminNoteAdviceLetter(ctx context.Context, input models.CreateTRBAdminNoteGuidanceLetterInput) (*models.TRBAdminNote, error) {
+// CreateTRBAdminNoteGuidanceLetter is the resolver for the createTRBAdminNoteGuidanceLetter field.
+func (r *mutationResolver) CreateTRBAdminNoteGuidanceLetter(ctx context.Context, input models.CreateTRBAdminNoteGuidanceLetterInput) (*models.TRBAdminNote, error) {
```

Comment:
for future reference, after renaming something in `schema.graphql`, renaming these functions using an IDE's refactor mechanism _before_ running `scripts/dev gql` avoids a `panic` placeholder being slotted in